### ISSUE TITLE
Driver '76 instability

### ISSUE THREAD

**Issue by hrydgard** [hrydgard/ppsspp#16904]
BLOC 15 on discord has been playing Driver '76 (ULES-00740), and encountered a hang right after this save state:

[Driver '76 ULES00740_1.01_0.zip](https://github.com/hrydgard/ppsspp/files/10581526/Driver.76.ULES00740_1.01_0.zip)

He also reports various random-feeling crashes.

**Comment by Panderner** [comment]
Still happens when simulate UMD delays is set?

**Comment by blocxv** [comment]
not work if i use this setting game freeze when i launch an savestate my trouble seem apear only in mission 3 when we collect 3 truck

**Comment by Gamemulatorer** [comment]
Related issue https://github.com/hrydgard/ppsspp/issues/12054

**Comment by sum2012** [comment]
the freeze problem should be sceAtrac problem.
29:25:305 ATRAC3 play  I[ME]: HW\MediaEngine.cpp:87 FF: JS mono Sound Unit id != 3.
29:25:305 ATRAC3 play  I[ME]: HW\MediaEngine.cpp:87 FF: Frame decoding error!
29:25:305 ATRAC3 play  E[ME]: HLE\sceAtrac.cpp:604 avcodec_decode_audio4: Error decoding audio -1094995529 / bebbb1b7
full log: https://gist.github.com/sum2012/4d1d9d6cdd95d813c8b1d9f2923fe8da

**Comment by sum2012** [comment]
crash full debug v1.16.5-83-gb85f7e28a log
[ppsspplog.zip](https://github.com/hrydgard/ppsspp/files/12781727/ppsspplog.zip)
using interpreter or UMD delay also not work

**Comment by sum2012** [comment]
I need a game save rather than save status to test

**Comment by sum2012** [comment]
I play the game to myself, after 1 stage, auto save ->
[ULES00740001.zip](https://github.com/hrydgard/ppsspp/files/13257937/ULES00740001.zip)

In mission 2 save status->
[ULES00740_1.01_2.zip](https://github.com/hrydgard/ppsspp/files/13257940/ULES00740_1.01_2.zip)

Keep drive in font,you will freeze a while then car graphic error ->
 
![2](https://github.com/hrydgard/ppsspp/assets/2177532/804bcec8-b944-417f-a9c4-65b06b313b25)
![3](https://github.com/hrydgard/ppsspp/assets/2177532/a054437d-ed1c-4b31-a5f1-a4ae7f88439e)
work around Change I/O timing method to "Simulate UMD delay" or "Simulate MD slow reading speed"

**Comment by sum2012** [comment]
v1.16.6-341-g4d7f00f96 remain Atrac freeze problem.
I need game save rather than save status to test.

**Comment by hrydgard** [comment]
I think it's likely that this is fixed with the new atrac impl - @sum2012 wanna test?

**Comment by sum2012** [comment]
I need game save rather than save status to test.
Anyway , The first save status do not seem atrac problem.
![Image](https://github.com/user-attachments/assets/23115b9e-53b2-41a5-a186-b621f6b8b446)

The second  save status do not solve.
![Image](https://github.com/user-attachments/assets/16b25578-bf56-422f-a562-69779162e067)

**Comment by hrydgard** [comment]
Right, can't test the new sceAtrac with save states, as it's impossible to switch - old states will keep using the old implementation.

**Comment by ppmeis** [comment]
Tested in latest build. I played about 3-4 missions and no issues.

**Comment by FATCatAndroid12** [comment]
v1.19 milestone
